docs/package.json

```diff
@@ -12,8 +12,9 @@
     "storybook": "start-storybook -p 6006",
     "format": "prettier --write '**/*.js'",
     "test": "jest",
-    "test:watch": "jest --watchAll --verbose",
-    "test:coverage": "jest --verbose --coverage",
+    "test:watch": "npm run test --watchAll --verbose",
+    "test:coverage": "npm run test --verbose --coverage",
```

Comment:
That's lacking the -- before the args, like you did in test:update. Consequence is the coverage won't be generated. Same applies to test:watch above.